src/lib.rs

```diff
@@ -152,9 +152,9 @@ impl Plugin for SteamworksPlugin {
             Ok((client, single)) => {
                 app.insert_resource(Client(client.clone()))
                     .insert_non_send_resource(single)
-                    .add_system_to_stage(
-                        CoreStage::First,
-                        run_steam_callbacks.label(SteamworksSystem::RunCallbacks),
+                    .add_system(run_steam_callbacks
+                        .in_base_set(CoreSet::First)
+                        .in_set(SteamworksSystem::RunCallbacks),
```

Comment:
```suggestion
                    .add_system(
                        run_steam_callbacks
                            .in_base_set(CoreSet::First)
                            .in_set(SteamworksSystem::RunCallbacks),
```
Aaah, I botched it. Try again :P ?